client/src/components/CuisineList.js

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { colors } from "../themes/theme";
+import CuisineTag from "./CuisineTag"
+import React from "react";
+
+const useStyles = makeStyles({
+    name:{
+        backgroundColor:colors.brandLight,
+        color:"white",
+    }
+});
+
+export default function Cuisinelist({ cuisineList }) {
+
+    const classes = useStyles();
+    const cuisinecomponent = cuisineList.map((name,i) =>{
```

Comment:
I suggest changing variable name for clarity
```suggestion
    const cuisineComponents = cuisineList.map((name,i) =>{
```